scripts/fit.py

```diff
@@ -236,29 +236,36 @@ def jac(x, pars):
         options = dict(maxiter=1e4, gtol=1e-3)
 
     def _fit(N_PARS=3, sign=1):
+        bounds = None
+        if method in ["Nelder-Mead"]:
+            bounds = ((-1e9, 1e9), (-1e9, 0), (0, 10))
         if N_PARS == 3:
             result = opt.minimize(
                 objective,
                 x0=(sign * 1, -1, 1),
-                bounds=None,
+                bounds=bounds,
                 tol=1e-6,
                 method=method,
                 jac=_jac,
                 hess=_hess,
                 options=options,
             )
         elif N_PARS == 2:
+            if method in ["Nelder-Mead"]:
+                bounds = ((-1e9, 1e9), (-1e9, 0))
             result = opt.minimize(
                 objective,
                 x0=(sign * 1, -1),
-                bounds=None,
+                bounds=bounds,
                 tol=1e-6,
                 method=method,
                 jac=_jac,
                 hess=_hess,
                 options=options,
             )
         elif N_PARS == 1:
+            if method in ["Nelder-Mead"]:
+                bounds = ((-1e9, 1e9), (0, 10))
```

Comment:
Shouldn't this case only have one parameter to fit? I see bounds for two, though.

Reply:
The fit function for N_PARS == 1 has two parameters:
```python
        elif N_PARS == 1:
            a, b = pars
            return a * x + b
```

Reply:
Which actually messes up the chi2/ndf calculation. This is a bug :D

Reply:
I will open a Bug report. Fixing this is out of scope for this PR.

Reply:
See #58 
Resolving this comment as this is out of scope for this PR.